documentation/api/query/v4/packages.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "PuppetDB 4.4: Packages endpoint"
+title: "PuppetDB 4.4: Package endpoints"
 layout: default
 canonical: "/puppetdb/latest/api/query/v4/packages.html"
```

Comment:
I'm not necessarily against squishing these into one doc, but I think it'll be the first time we do this. Things like line 4 here make me wonder if the presentation will make sense on the site.